apps/aecore/lib/aecore/contract/tx/contract_call_tx.ex

```diff
@@ -0,0 +1,377 @@
+ defmodule Aecore.Contract.Tx.ContractCallTx do
+   @moduledoc """
+   Aecore structure for Contract Call
+   """
+
+   @behaviour Aecore.Tx.Transaction
+
+   alias __MODULE__
+   alias Aecore.Account.{Account, AccountStateTree}
+   alias Aecore.Tx.DataTx
+   alias Aecore.Chain.Worker, as: Chain
+   alias Aecore.Chain.{Identifier, Chainstate}
+   alias Aecore.Contract.{Contract, Call, CallStateTree, Dispatch, ContractStateTree}
+   alias Aecore.Tx.Transaction
+
+   @version 1
+
+   @type id :: binary()
+
+   @type payload :: %{
+           contract: Identifier.t(),
```

Comment:
As I understand key `contract:` can be `binary` as well. Is it right?

Reply:
It should be an Identifier at all times.